README.md

```diff
@@ -1,99 +1,91 @@
-# Deploy to iTunes Connect - Application Loader
```

Comment:
I would change the order and put the `(formerly iTunes Connect)` behind the `App Store Connect` text because (I kid you not) for a brief second I started to think that the `Application Loader` was renamed? 😂

So my suggestion:
```
Deploy to App Store Connect (formerly iTunes Connect) - Application Loader
```

And the same for in the `step.yml` too.

Reply:
My problem with this variant is that the most important part (Application loader) will be cut off in the workflow editor, and both Deploy to App Store Connect steps will look the same